src/nl/hannahsten/texifyidea/lang/DefaultEnvironment.kt

```diff
@@ -81,8 +81,10 @@ enum class DefaultEnvironment(
     GATHER(environmentName = "gather", context = Context.MATH, dependency = AMSMATH),
     GATHERED(environmentName = "gathered", context = Context.MATH, dependency = AMSMATH),
     GATHER_STAR(environmentName = "gather*", context = Context.MATH, dependency = AMSMATH),
+    MATRIX(environmentName = "matrix", context = Context.MATH, dependency = AMSMATH),
     MULTLINE(environmentName = "multline", context = Context.MATH, dependency = AMSMATH),
     MULTLINE_STAR(environmentName = "multline*", context = Context.MATH, dependency = AMSMATH),
+    PMATRIX(environmentName = "pmatrix", context = Context.MATH, dependency = AMSMATH),
```

Comment:
I think I would prefer that the other matrices are in the autocomplete as well (I am an avid user of `bmatrix`)

Reply:
bmatrix was already part of the list. only pmatrix and matrix were missing. Added all the other stuff from the other reply as well.